### DISCUSSION TITLE
CSV upload with Location Notes entry is not updated

### DISCUSSION THREAD

**Discussion by DCO84770** [code4recovery/12-step-meeting-list#1409] [Q&A] [answered]
Hello
All Springdale meetings are here:
https://dixieaaoffice.com/meetings/?region=springdale
I added a new meeting for 2 pm Wednesday and entered Auxiliary Room in Location Notes and Uploaded the file. Repeated attempts resulted in that field inheriting another meeting's location note as Library Room. I can change it in the web interface to Auxiliary Room, but then all in-person meetings show as Auxiliary Room. 
This seems like a bug to me as I would think different meetings ought to have different meeting notes if needed. IE different rooms at the same street address.

If this operation is normal, can anyone advise a workaround?
Thanks
Roger

**Comment by joshreisner** [comment, marked as answer]
hi @DCO84770 the location name and notes fields are shared by all meetings at a given location, which is determined by address. this is a little clearer when entering information manually on the edit screen. we do get people confused about this from time to time if they are importing their changes from a spreadsheet, since it doesn't follow those rules

if the info needs to vary, such as a room number that is not shared by the other meetings at that address, then it should go in the notes field

**Reply by DCO84770** [reply]
Thank you!
